jjb/jobs/qcs-PILOT-nightly-automation.yaml

```diff
@@ -0,0 +1,87 @@
+- job:
+    name: 'qcs-PILOT-nightly-automation'
+    node: 'f27-os'
+    triggers:
+        - timed: '@midnight'
+    scm:
+        - git:
+            url: https://github.com/quipucords/camayoc.git
+            basedir: camayoc
+            branches:
+                - '*/quipucords/pilot'
+            skip-tag: true
+    wrappers:
+        - credentials-binding:
+            - file:
+                credential-id: 4c692211-c5e1-4354-8e1b-b9d0276c29d9
+                variable: ID_JENKINS_RSA
+        - ssh-agent-credentials:
+            users:
+                - '390bdc1f-73c6-457e-81de-9e794478e0e'
+    builders:
+        - config-file-provider:
+            files:
+              - file-id: '62cf0ccc-220e-4177-9eab-f39701bff8d7'
+                target: 'camayoc/config.yaml'
+        - copyartifact:
+                project: qpc_docker_beta
+                filter: quipucords.*.tar.gz
+                target: ${WORKSPACE}/container/
+                stable: true
+
+        - shining-panda:
+            build-environment: virtualenv
+            python-version: System-CPython-3.6
+            nature: shell
+            command: |
+                export XDG_CONFIG_HOME=$PWD
+                mkdir -p /home/jenkins/.ssh
+                cp "${ID_JENKINS_RSA}" /home/jenkins/.ssh/id_rsa
+                chmod 0600 /home/jenkins/.ssh/id_rsa
+                
+                # rename container to something without long hash
+                mv ${WORKSPACE}/container/quipucords* quipucords.pilot.tar.gz
+
+                sed -i "s/{jenkins_slave_ip}/${OPENSTACK_PUBLIC_IP}/" camayoc/config.yaml
+
+                echo "OPTIONS=--log-driver=journald" > docker.conf
+                echo "DOCKER_CERT_PATH=/etc/docker" >> docker.conf
+                sudo cp docker.conf /etc/sysconfig/docker
+                sudo systemctl start docker
+                echo "load docker container from tarball"
+                sudo docker load -i ${WORKSPACE}/quipucords.pilot.tar.gz
+
+                # make log dir to save server logs
+                mkdir ${WORKSPACE}/log
+                # run docker container, mounting local volumes for logs and ssh keys 
+                sudo docker run -d -p "443:443" -v /tmp:/tmp -v /home/jenkins/.ssh:/home/jenkins/.ssh -v ${WORKSPACE}/log:/var/log -i "quipucords:pilot"
```

Comment:
Did you mean for `/tmp` to map to `/var/data`?
Did you mean for `/home/jenkins/.ssh` to map to `/sshkeys`?

Reply:
Both are what @elyezer had decided on in the past. The sshkey dir simplifies our config file situation, so we can share it across jobs and not have to make up for the fact that the path is different inside the container.

I can adjust the mapping of the tmp dir because I think you might be right about that...wanting the database to end up in tmp.